Lib/test/test_asyncio/test_locks.py

```diff
@@ -305,6 +305,12 @@ def test_context_manager(self):
 
         self.loop.run_until_complete(f())
 
+    def test_lock_loop_not_running(self):
+        loop = asyncio.new_event_loop()
```

Comment:
Forgot to mention this in the review, but this line isn't needed either:

```suggestion
```

Reply:
If I let just the
```
with self.assertWarns(DeprecationWarning):
    asyncio.Queue()
```

I have a Runtime Error: `RuntimeError: There is no current event loop in thread 'MainThread'.
`

Reply:
You can resolve that by creating an event loop in the unit test class's `setUp()` method, setting it as the event loop, and the removing it in the `tearDown()` method. Something like this should work:

```py

    def setUp(self):
        self.loop = asyncio.new_event_loop()
        asyncio.set_event_loop(self.loop)

    def tearDown(self):
        self.loop.close()
        self.loop = None
```